routes/web.php

```diff
@@ -17,48 +18,51 @@
-Route::delete('organizations/{organization}')->name('organizations.destroy')->uses('OrganizationsController@destroy')->middleware('auth');
-Route::put('organizations/{organization}/restore')->name('organizations.restore')->uses('OrganizationsController@restore')->middleware('auth');
+    // Organizations
+    Route::get('organizations')->name('organizations')->uses('OrganizationsController@index')->middleware('remember');
+    Route::get('organizations/create')->name('organizations.create')->uses('OrganizationsController@create');
+    Route::post('organizations')->name('organizations.store')->uses('OrganizationsController@store');
+    Route::get('organizations/{organization}/edit')->name('organizations.edit')->uses('OrganizationsController@edit');
+    Route::put('organizations/{organization}')->name('organizations.update')->uses('OrganizationsController@update');
+    Route::delete('organizations/{organization}')->name('organizations.destroy')->uses('OrganizationsController@destroy');
+    Route::put('organizations/{organization}/restore')->name('organizations.restore')->uses('OrganizationsController@restore');
+
+    // Contacts
+    Route::get('contacts')->name('contacts')->uses('ContactsController@index')->middleware('remember');
+    Route::get('contacts/create')->name('contacts.create')->uses('ContactsController@create');
+    Route::post('contacts')->name('contacts.store')->uses('ContactsController@store');
+    Route::get('contacts/{contact}/edit')->name('contacts.edit')->uses('ContactsController@edit');
+    Route::put('contacts/{contact}')->name('contacts.update')->uses('ContactsController@update');
+    Route::delete('contacts/{contact}')->name('contacts.destroy')->uses('ContactsController@destroy');
+    Route::put('contacts/{contact}/restore')->name('contacts.restore')->uses('ContactsController@restore');
+});
 
-// Contacts
-Route::get('contacts')->name('contacts')->uses('ContactsController@index')->middleware('remember', 'auth');
-Route::get('contacts/create')->name('contacts.create')->uses('ContactsController@create')->middleware('auth');
-Route::post('contacts')->name('contacts.store')->uses('ContactsController@store')->middleware('auth');
-Route::get('contacts/{contact}/edit')->name('contacts.edit')->uses('ContactsController@edit')->middleware('auth');
-Route::put('contacts/{contact}')->name('contacts.update')->uses('ContactsController@update')->middleware('auth');
-Route::delete('contacts/{contact}')->name('contacts.destroy')->uses('ContactsController@destroy')->middleware('auth');
-Route::put('contacts/{contact}/restore')->name('contacts.restore')->uses('ContactsController@restore')->middleware('auth');
 
 // Reports
-Route::get('reports')->name('reports')->uses('ReportsController')->middleware('auth');
+Route::get('reports')->name('reports')->uses('ReportsController');
 
 // 500 error
 Route::get('500', function () {
-    echo $fail;
```

Comment:
This is an intentional bug...to illustrate how error handling is done in Inertia. Please revert change.

Reply:
done